### PR TITLE
Feature/web nav

### PR DESCRIPTION
**Pull request by toTOEro**

### CODE REVIEW

File: views/home.handlebars [hunk 1]
```diff
@@ -1,79 +1,40 @@
-<div class="login-card">
```
Comment (margotcooper) [inline views/home.handlebars:1]: Hey. So if I push your changes through, everything I added will be deleted... is that what we want? @toTOEro

Comment (toTOEro) [reply views/home.handlebars:1]: It's moved to login.handlebars; that will let it render properly

File: views/layouts/main.handlebars [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
   <title>Medi</title>
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8"
+    crossorigin="anonymous"></script>
   <link rel="stylesheet" href="/css/jass.css">
   <link rel="stylesheet" href="/css/style.css">
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
```
Comment (margotcooper) [inline views/layouts/main.handlebars:6]: It is ok to delete this link here, because you included it above on lines 5-6 @toTOEro

**Review by margotcooper — APPROVED** [review]
Ignore my last comments! This looks great, thanks for moving it for me. Sorry about the confusion!

File: views/layouts/main.handlebars (hunk 2 above)
Comment (toTOEro) [reply views/layouts/main.handlebars:6]: yup, want to load bootstrap styles then custom styles

**Pull request merged**